### DISCUSSION TITLE
Theme colors don't work well on some parts of typed code

### DISCUSSION THREAD

**Discussion by lrtf** [dracula/dracula-theme#796] [General] [answered]
View a snippet of JavaScript code

![image](https://user-images.githubusercontent.com/23635488/210072030-90f4fe17-9948-4d2f-8c35-01b8e1b356ab.png)

View a snippet of Typescript code

![image](https://user-images.githubusercontent.com/23635488/210072093-91cf6a92-65f3-4d88-b895-738a2af49e9c.png)

the constant name, this and the repository have the same color

Is there a way to configure for typescript?

Informations:
Extension: Dracula Official v2.24.2
Editor: Visual Code Studio
System: Windows 10

**Comment by luxonauta** [comment, marked as answer]
Hi 👋🏻, thank you for your interest in providing this feedback. ✨

But there is a better place to share the same, which is the project repository, where you can talk to the theme maintainers.

✅ [Clicking here](https://github.com/dracula/visual-studio-code/issues/new), you can open an Issue on it.
